doc/source/notebooks/advanced/varying_noise.pct.py

```diff
@@ -132,7 +139,7 @@ def objective_closure():
 
 plt.errorbar(X.squeeze(), Y.squeeze(), yerr=2*(np.sqrt(NoiseVar)).squeeze(),
              marker='x', lw=0, elinewidth=1., color='C1')
-plt.xlim(-5, 5);
+plt.xlim(-5, 5)
```

Comment:
The ; at the end was intentional so that jupyter doesn't print the return value
```suggestion
plt.xlim(-5, 5);
```

Reply:
:sad face: ok. will revert